examples/dam-app/src/index.js

```diff
@@ -0,0 +1,110 @@
+import './index.css';
+import { setup } from '@contentful/dam-app-base';
+import { EntityList } from '@contentful/f36-components';
+import pick from 'lodash/pick';
+import { useAsync } from 'react-async-hook';
+import { render } from 'react-dom';
+
+const CTA = 'Sample DAM Demo App';
+const FIELDS_TO_PERSIST = ['id', 'name', 'url'];
+
+setup({
+  cta: CTA,
+  name: 'Contentful DAM Demo App',
+  logo: 'https://images.ctfassets.net/fo9twyrwpveg/6eVeSgMr2EsEGiGc208c6M/f6d9ff47d8d26b3b238c6272a40d3a99/contentful-logo.png',
+  color: '#036FE3',
+  description:
+    'This is a sample Application to demonstrate how to make a custom DAM application on top of Contentful.',
```

Comment:
Maybe adding a "DAM (Digital Asset Management)" would not hurt here wdyt?

Reply:
Will add